pkgs/by-name/bi/bitwarden-desktop/package.nix

```
@@ -90,7 +90,7 @@ in buildNpmPackage rec {
     moreutils
     napi-rs-cli
     pkg-config
-    python3
+    python311
```

Does
```suggestion
    (python3.withPackages (ps: with ps; [ setuptools ]))
```
work with Python 3.12?